09-intermediate-rails/creating-apis.md

```diff
@@ -62,30 +62,34 @@ You'll notice that we have one basic get route created for pets, and one basic c
 The error below is where we diverge from our "normal" rails approach. Normally, we'd create a view that would correspond with our controller action.  The error should be something like:
 
 ```bash
-Missing template pets/index, application/index...
+Missing a template for this request format and variant.
 ```
 
 Since we are building a JSON API, we don't want to render an HTML template (or really use any of the view layer). Instead we want to return the requested data in a standard format. Using the `render` method in the controller with some new options, we can choose to return json:
 
 ```ruby
 # pets_controller.rb
 def index
-  render :json => { ready_for_lunch: "yassss" }
+  render json: { ready_for_lunch: "yassss" }
 end
 ```
 
-Notice that we didn't for realsies write any JSON. We provided a plain Ruby hash and let Rails do the conversion for us (with the `render :json =>` call. So to make progress on our tests, we could do something like:
+Notice that we didn't for realsies write any JSON. We provided a plain Ruby hash and let Rails do the conversion for us (with the `render json: ` call. When we run `rails test` again we see that our test is expecting an `Array` rather than a Ruby `Hash` which is what our controller method contains.
+
+To create the appropriate data and ensure we send back an array, we could do something like:
```

Comment:
3 things:

1. You have extra whitespace on `render json: `
1. The following sentence ("When we run...") should probably be a new paragraph.
1. I'm not sure I understand the last sentence - maybe reword it, or at least use a different verb than "contains"